binderhub/static/js/index.js

```diff
@@ -76,7 +76,7 @@ function updateRepoText() {
   }
   else if (provider === "git") {
     text = "Arbitrary git repository URL (http://git.example.com/repo)";
-    tag_text = "Git commit SHA";
+    tag_text = "Git branch, tag, or commit SHA";
```

Comment:
Should be just `"Git branch, tag, or commit"` to match other similar providers exactly.

Reply:
It doesn't work for partial commit strings, that's why I explicitly added 'SHA'.

Reply:
:+1: